container.go

```diff
@@ -101,28 +102,29 @@ type ContainerRequest struct {
 	Cmd             []string
 	Labels          map[string]string
 	Mounts          ContainerMounts
-	Tmpfs           map[string]string
+	Tmpfs           map[string]string // Deprecated: Use PreCreationHook instead
```

Comment:
In Java, we expose this in the 1st level API. Why? Because it is a popular choice for database containers to get some quick performance wins.